day_two_guide/topics/network_connectivity.adoc

```diff
@@ -83,6 +83,20 @@ $ curl -k https://master.example.com:443/healthz
 ok
 ----
 
+*Determining internal and external master URLs*
+
+The following commands can be used to determine the URLs that are used by the internal cluster and external clients. Example URLs are found in the previous example.
+
+To determine the Internal cluster URL:
```

Comment:
Does "Internal" and "External" (below) really have to be capitalized, or can they be lowered?